migrator/README.md

```diff
@@ -0,0 +1,268 @@
+# Migrator Library
+
+The library takes in E.164 phone numbers with their corresponding BCP-47 country code. If inputted numbers match any of the supported internal renumbering ‘recipes’, which are created from official country documentation, said numbers will undergo a process of migration to convert them into the most up to date, dialable format. A given “recipe” is an internal record which holds data needed to migrate a stale number into one that is valid and dialable. 
```

Comment:
Can you link `BCP-47 country code` to a definition? I'm guessing most users might not know what that is.